examples/sentiment/scripts/gpt2-sentiment.py

```diff
@@ -67,6 +67,7 @@ class ScriptArguments:
     )
     early_stopping: Optional[bool] = field(default=False, metadata={"help": "whether to early stop"})
     target_kl: Optional[float] = field(default=0.1, metadata={"help": "kl target for early stopping"})
+    loss_spike_threshold: Optional[float] = field(default=1000.0, metadata={"help": "kl target for early stopping"})
```

Comment:
I think the metadata needs to be updated :)

Reply:
Are the ratios not clipped with ppo_clip?